From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Fri, 30 Dec 2022 18:51:23 +0100
Subject: docs: Exclude some not available Sphinx extensions

sphinxemoji.sphinxemoji, sphinx-jsonschema and sphinxarg.ext are not
packaged and can't be used due this.

Forwarded: Not-Needed
---
 docs/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 185b03f..87e92e3 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -75,9 +75,9 @@ extensions = [
     "sphinx.ext.napoleon",
     "sphinx.ext.extlinks",
     "sphinx_copybutton",
-    "sphinxemoji.sphinxemoji",
-    "sphinx-jsonschema",
-    "sphinxarg.ext",
+#    "sphinxemoji.sphinxemoji",
+#    "sphinx-jsonschema",
+#    "sphinxarg.ext",
 ]
 
 # ----------------------------------
